docs/helm-chart/index.rst

```diff
@@ -119,10 +119,10 @@ To uninstall/delete the ``airflow`` deployment:
 
 The command removes all the Kubernetes components associated with the chart and deletes the release.
 
-Installing the Chart with ArgoCD
---------------------------------
+Installing the Chart with ArgoCD or FluxCD
+------------------------------------------
```

Comment:
```suggestion
Installing the Chart with Argo CD or Flux
-----------------------------------------
```

Just from looking at their site, they seem to just use "Flux", not "FluxCD"?
Also looks like it should be "Argo CD" instead too.

Reply:
Agreed. They used to use FluxCD in the beginning.